Examples/2048/2048TV.xcodeproj/project.pbxproj

```diff
@@ -0,0 +1,429 @@
+// !$*UTF8*$!
+{
```

Comment:
Yeah, the goal is to have proj files generated by Buck.
We'll be working on this in the following months